src/Connections/SetupConnection.ts

```diff
@@ -216,7 +216,12 @@ export class SetupConnection extends CommandConnection {
         this.pushConnections(c.connection);
         const url = new URL(c.connection.hookId, this.config.generic.parsedUrlPrefix);
         const adminRoom = await this.getOrCreateAdminRoom(userId);
-        await adminRoom.sendNotice(`You have bridged a webhook. Please configure your webhook source to use ${url}.`);
+        await adminRoom.sendNotice(
+            `You have bridged the webhook "${name}" in https://matrix.to/#/${this.roomId} .\n` +
```

Comment:
While an official Synapse will generate safe room IDs, the spec doesn't seem to limit what characters might be included.

matrix.to can deal with URL-encoded room IDs and room aliases.

```suggestion
            `You have bridged the webhook "${name}" in https://matrix.to/#/${encodeURIComponent(this.roomId)} .\n` +
```

Alternatively, if you think it looks too weird that ":" gets encoded:
```suggestion
            `You have bridged the webhook "${name}" in https://matrix.to/#/${encodeURIComponent(this.roomId).replace(/%3A/g, ":")} .\n` +
```

Reply:
Former looks fine to me :)